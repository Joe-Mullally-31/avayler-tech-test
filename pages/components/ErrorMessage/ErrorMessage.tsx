import styles from '../../../styles/ErrorMessage.module.css';

export const ErrorMessage = () => (
  <div className={styles.errorMessageContainer} role="alert" aria-label="Error loading launch information.">
    <h2>There was an error loading SpaceX information!</h2>
    <p>
      Please check your internet connection and try again later. Alternatively{' '}
      <a className={styles.supportLink} href="mailto: support@mail.co">
        contact support
      </a>
      .
    </p>
  </div>
);
