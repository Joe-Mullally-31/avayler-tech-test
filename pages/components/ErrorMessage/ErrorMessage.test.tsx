import { render } from '@testing-library/react';
import { ErrorMessage } from './ErrorMessage';

it('should render as expected', () => {
  expect(render(<ErrorMessage />)).toMatchSnapshot();
});
