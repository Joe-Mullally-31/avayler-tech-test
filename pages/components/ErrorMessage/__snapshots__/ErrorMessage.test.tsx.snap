// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        aria-label="Error loading launch information."
        class="errorMessageContainer"
        role="alert"
      >
        <h2>
          There was an error loading SpaceX information!
        </h2>
        <p>
          Please check your internet connection and try again later. Alternatively
           
          <a
            class="supportLink"
            href="mailto: support@mail.co"
          >
            contact support
          </a>
          .
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      aria-label="Error loading launch information."
      class="errorMessageContainer"
      role="alert"
    >
      <h2>
        There was an error loading SpaceX information!
      </h2>
      <p>
        Please check your internet connection and try again later. Alternatively
         
        <a
          class="supportLink"
          href="mailto: support@mail.co"
        >
          contact support
        </a>
        .
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
